homework_create_more_models/README.md

```diff
@@ -287,7 +287,7 @@ def comment_remove(request, pk):
     comment = get_object_or_404(Comment, pk=pk)
     post_pk = comment.post.pk
     comment.delete()
-    return redirect('post_detail', pk=post_pk)
+    return redirect('post_detail', pk=comment.post.pk)
```

Comment:
Hi @tfevan - thank you for contributing!

There shouldn't be any error from the current version since `post_pk` is defined by the second line in that function.

However, either version would be fine - yours is probably better as it's more consistent with the function just above it. Since we're not going to be using `post_pk` in `comment_remove` any more, please can you remove the line that assigns to it?

Thanks!